### PR TITLE
394 aerogear sdk make docs more verbose, 537 add note where are aerogear sdk packages hosted 

### PR DESCRIPTION
**Pull request by pmacko1**
https://github.com/aerogear/mobile-docs/issues/394

### CODE REVIEW

File: modules/ROOT/pages/_partials/con_aerogear-sdk.adoc [hunk 1]
```diff
@@ -1,11 +1,16 @@
-= Introduction to the AeroGear SDK
+= AeroGear SDK
 
 After you have xref:getting-started.adoc#registering[registered] a {mobile-client}, xref:getting-started.adoc#binding[bound] the {mobile-client} to a mobile service, and xref:getting-started.adoc#downloading-the-mobile-services-configuration-file[downloaded] the mobile services configuration file, you can start developing your mobile app using mobile services.
 
 The {org-name} SDK supports the following platforms:
```
Comment (finp) [inline modules/ROOT/pages/_partials/con_aerogear-sdk.adoc]: mobile development platforms

File: modules/ROOT/pages/_partials/con_aerogear-sdk.adoc [hunk 2]
```diff
@@ -1,11 +1,16 @@
...
 
+* xref:android[Android]
+* xref:ios[iOS]
+* xref:cordova[Cordova]
+* xref:xamarin[Xamarin]
+
 [[android]]
-== Android
+== Implementing AeroGear SDK into Android apps
```
Comment (finp) [inline modules/ROOT/pages/_partials/con_aerogear-sdk.adoc]: Using the {org-name} SDK to develop ...

File: modules/ROOT/pages/_partials/con_aerogear-sdk.adoc (hunk 1 above)
Comment (pmacko1) [reply modules/ROOT/pages/_partials/con_aerogear-sdk.adoc]: mobile platforms

**Comment by pmacko1** [comment]
@finp I think this one has been completed?

**Pull request closed without merging**